modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/filters/components/Configuration.tsx

```diff
@@ -13,7 +13,7 @@ import {InputLocalized} from 'frontend-js-components-web';
 import {openModal} from 'frontend-js-web';
 import React, {useState} from 'react';
 
-import FieldSelectModalContent from '../../../components/FieldSelectModalContent';
+import FieldSelectModalContent from '../../../components/AddDataSourceFieldsModalContent';
```

Comment:
this should look like:
`import AddDataSourceFieldsModalContent from '../../../components/AddDataSourceFieldsModalContent';`

and so the below variable `openSelectFieldModal` needs renaming

Reply:
Yep, I've renamed it